settings_menu/PlayerPrefs.cpp

```diff
@@ -171,6 +173,12 @@ void PlayerPrefs::privateSetPlayerName(const std::string& p_name)
 {
 	m_playerName = p_name;
 	m_hasUnsavedChanges = true;
+
+	GameModeManager::getInstance()->setPointTextBoxes();
```

Comment:
Does this break stuff if you set your name on the main menu?

Reply:
Oh shoot, yes probably. Let me take a look and change it

Reply:
Sorry for delay, was on a different branch and had to rebuild.

Changed it to an event so it would only set the textboxes when the manager has been created. In the process of testing it now but it had to rebuild again 😢

Reply:
Adjusted